.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
   test:
     name: ci ${{ matrix.version }} - ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 30
+    timeout-minutes: 45
```

Comment:
The new test in the GHA CI takes ~15 minutes, and the CI takes ~35 minutes in total

Reply:
Maybe we can split this off to a second action so that it can run in parallel?